packages/features/bookings/Booker/components/Header.tsx

```diff
@@ -54,12 +62,23 @@ export function Header({
       <LayoutToggle onLayoutToggle={onLayoutToggle} layout={layout} enabledLayouts={enabledLayouts} />
     );
   };
+  const isMyLink = username === session?.data?.user.username; // TODO: check for if the user is the owner of the link
 
   // In month view we only show the layout toggle.
   if (isMonthView) {
     return (
       <div className="flex gap-2">
         <OverlayCalendarContainer />
+        {isMyLink && (
+          <Tooltip content={t("troubleshooter_tooltip")} side="bottom">
+            <Button
+              color="primary"
+              target="_blank"
+              href={`${WEBAPP_URL}/availability/troubleshoot?eventType=${eventSlug}`}>
```

Comment:
this works, but for some reason the URL does not take the state and by default loads the first event-type in the dropdown

cc @sean-brydon